packages/react-core/src/components/ApplicationLauncher/__tests__/__snapshots__/ApplicationLauncher.test.tsx.snap

```diff
@@ -121,7 +120,7 @@ exports[`ApplicationLauncher custom icon 1`] = `
       aria-label="Application launcher"
       className="pf-c-app-launcher pf-m-expanded"
       data-ouia-component-id={5}
-      data-ouia-component-type="PF4/Dropdown"
+      data-ouia-component-type="PF4/undefined"
```

Comment:
@redallen These are a result of above I think

Reply:
It doesn't look like you modified `ApplicationLauncher.tsx`. Make sure you pass the OUIA props to its `DropdownContext.Provider`.